apps/bsd/README.md

```diff
@@ -28,17 +28,19 @@ yarn install
 
 ## Run
 
-1. Start [`module-scan`](https://github.com/votingworks/module-scan/).
+1. Start [`module-scan`](https://github.com/votingworks/vxsuite/tree/main/apps/module-scan).
 2. Start
-   [`module-smartcards`](https://github.com/votingworks/module-smartcards/).
+   [`module-smartcards`](https://github.com/votingworks/vxsuite/tree/main/apps/module-smartcards).
 3. Start the app:
 
    ```
    yarn start
    ```
 
+To set the election configuration you will either need to scan a smartcard (you can use the mockCardReader script in module-smartcards for this), load an election.json file, or load a ballot export zip file. You should load a ballot export zip if you intend to test actually scanning images.
+
 To display a batch of scanned ballots, add ballot images into the
-`module-scan/ballot-images` directory, then click "Scan New Batch" button.
+`module-scan/ballot-images` directory, then click "Scan New Batch" button. You can also start module-scan with the MOCK_SCANNER_FILES environment variable set as described in [`module-scan`](https://github.com/votingworks/vxsuite/tree/main/apps/module-scan).
```

Comment:
What this said before is actually not quite true anymore. I'd change this to just tell people to look at the `module-scan` README, as you did.